app.js

```diff
@@ -10,17 +10,32 @@ const {
   messageHandler,
   logDeletedMessages,
 } = require('./handlers/messageHandlers');
+const fs = require('fs');
 
 require('dotenv').config();
 
 const client = getClient();
-const commandsDir = `${__dirname}/commands`;
 
-collectCommands(client, commandsDir);
-
-client.on('ready', () => {
-  console.log(`Logged in as ${client.user.tag}!`);
-});
+// Load commands
+const regularCommandsDir = `${__dirname}/commands`;
+const slashCommandsDir = `${__dirname}/slash-commands`;
+collectCommands({client, regularCommandsDir, slashCommandsDir});
+
+// Load events
+const eventsDir = `${__dirname}/events`;
+
+const eventFiles = fs
+  .readdirSync(eventsDir)
+  .filter((file) => file.endsWith('.js'));
+
+for (const file of eventFiles) {
+  const event = require(`${eventsDir}/${file}`);
+  if (event.once) {
+    client.once(event.name, (...args) => event.execute(...args));
+  } else {
+    client.on(event.name, (...args) => event.execute(...args));
+  }
+}
```

Comment:
Not a huge fan of having all this in `app.js`, but it's not a big deal. We can look into refactoring in a later PR.

Reply:
Yes, I agree that this loading logic can be moved out from app.js. That would probably include deciding if we want to keep the events in separate files or not and applying it consistently. I think that keeping them in individual files would be best.

Reply:
I moved all the events loading logic into config/loadEvents.js in a new commit to this PR.